.github/workflows/v1beta.yml

```diff
@@ -44,7 +46,7 @@ jobs:
           ./scripts/docker-deployment-sim.sh
         
       - name: Container Images Deployment
-        #if: github.event.pull_request.merged == true
+        if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
```

Comment:
update the condition to-
```yaml
if: github.event_name == 'push'
```
this is to ensure the pull request is merged, and to trigger the main branch push.